Simulator/Core/Driver.cpp

```diff
@@ -56,38 +49,88 @@
 
 using namespace std;
 
-// forward declarations
-bool parseCommandLine(int argc, char *argv[]);
-bool deserializeSynapses();
-void serializeSynapses();
+///  Handles parsing of the command line
+///
+///  @param  argc      argument count.
+///  @param  argv      arguments.
+///  @returns    true if successful, false otherwise.
+bool parseCommandLine(string cmdLineArguments)
```

Comment:
Let's make this a private method.

Reply:
Since the `parseCommandLine ` method need not be referenced outside the `Core.cpp` file (formerly known as `Driver.cpp`), should we give it an internal linkage by declaring it static? - then we don't have to add the private declaration in the header file?

Reply:
Well, this is an interesting question. The use of `static` you indicate is the C usage (and also for ordinary functions). If we make it a private method, then `static` means something different: that it can be called without making an instance of the class. Since we just want to make it executable only by other `Core` methods, that's not necessary.

Reply:
Thank you for clarifying that Professor